### DISCUSSION TITLE
👋

### DISCUSSION THREAD

**Discussion by axelboc** [ljosberinn/eslint-config-galex#779] [General]
Hi @ljosberinn, hope you're well!

Just wondering if you're planning any work on Galex in the near future. The config's dependencies are starting to fall out of date a fair bit, notably Typescript (5.1, when the latest is 5.4) and `@typescript-eslint/eslint-plugin` (5.56 when the latest is 7.8!)

> `@typescript-eslint/eslint-plugin` doesn't yet support ESLint v9, but it should be coming soon. Regardless, the latest v7 major seems like an important stepping stone.

FYI, I've upgraded TypeScript in my project, and this caused issues with [some rules from `eslint-plugin-etc`](https://github.com/cartant/eslint-plugin-etc/issues/64). I had to turn off the following: `etc/no-internal`, `etc/no-misused-generics` and `etc/no-deprecated` (I had enabled the latter myself, since `tsc` doesn't warn on deprecated APIs).

Note also that the current version of `@typescript-eslint/eslint-plugin` still logs the following warning every time I run ESLint, because the version of TypeScript is greater than 5.0: "WARNING: You are currently running a version of TypeScript which is not officially supported by @typescript-eslint/typescript-estree."

**Comment by ljosberinn** [comment]
Hey! I've been planning on doing an overhaul but I'm afraid I currently and for the near-future don't have the time available to work on it. I had been holding off updating it for a bit due to Flat Config and wait and see a bit how the ecosystem reacts/catches up.

What needs to be done is now summarized [here](https://github.com/ljosberinn/eslint-config-galex/issues/780)

**Reply by axelboc** [reply]
Awesome to hear, thanks for the update 😊

**Comment by ljosberinn** [comment]
@axelboc  It's been five months and while its been on the back of my mind, I definitely haven't gotten around to even looking at it and am afraid I won't in the near future. As a result I'm toying around the idea of archiving it...

**Reply by axelboc** [reply]
Fair enough, I understand. It would probably be easier to start from scratch anyway. Thanks for maintaining it these past few years, it's been a tremendous help. If you change your mind, let me know!

**Reply by ljosberinn** [reply]
That's my impression too haha which as you can imagine likely comes with even more work. Thank you and I will!
